{"sid": "674f768bd6b544c99fc9f4ff522e809e", "uid": null, "qid": "0af2e750ed5e48bba77e9ded65766b8b", "rid": null, "bars": {"default": {"percent": "100", "remaining": "0"}}, "messages": [{"type": "ERROR", "content": "process - failed executing: [0af2e750ed5e48bba77e9ded65766b8b]\nTraceback (most recent call last):\n  File \"/pysdk/.venv/lib/python3.8/site-packages/openfabric_pysdk/app/execution/action_dispatcher.py\", line 139, in process\n    output = self.workerContext.execution.execute(data, self.ray, self.state, update)\n  File \"/pysdk/.venv/lib/python3.8/site-packages/openfabric_pysdk/app/execution/application_interface.py\", line 66, in execute\n    self.execution_function(model)\n  File \"/openfabric/app/app/main.py\", line 103, in execute\n    msg , img , obj , sessid  = agent.Exec(user_prompt)\n  File \"/openfabric/app/app/Agent/Agent.py\", line 71, in Exec\n    return self.EXIT()\n  File \"/openfabric/app/app/Agent/Agent.py\", line 96, in EXIT\n    self.save_session()\n  File \"/openfabric/app/app/Agent/Agent.py\", line 108, in save_session\n    self.set_session_history(session_history)\n  File \"/openfabric/app/app/Agent/Agent.py\", line 53, in set_session_history\n    Agent._sessionsManager.set_session_history(self.session_id ,history)\nAttributeError: 'Agent' object has no attribute 'session_id'\n", "created_at": "2025-06-30T14:21:47.572018"}], "status": "FAILED", "created_at": "2025-06-30T14:20:53.622209", "updated_at": "2025-06-30T14:21:47.572054", "finished": true}